README.md

```diff
@@ -56,6 +59,22 @@ Access web locally:
 Check `Makefile` for shell commands if you want to run them with modified
 parameters.
 
+### Local .env file
+
+Creating a local `.env` file (see [Docker docs](https://docs.docker.com/compose/env-file/)
+on `.env` files) is necessary for some components in the app to work correctly,
+a basic example of such file contains the following:
+
+```
+ZOO_DEBUG=1
+ZOO_GITHUB_TOKEN=...
+ZOO_GITLAB_TOKEN=...
+ZOO_GITLAB_URL=https://gitlab.com
+```
+
+Version control tokens only need read access to repositories. 
+
+
```

Comment:
Please remove this blank line, coala is failing because of that.

Reply:
done